docs/06-authentication.md

```diff
@@ -0,0 +1,93 @@
+# Configuring Authentication
+OpenShift 4 installs with only a single cluster superuser, `kubeadmin`. If
+you want additional users to be able to authenticate to and use the cluster,
+you must configure an authentication provider. We will configure `htpasswd`
+authentication as an example.
+
+## Create the htpasswd file
+You can create an `htpasswd` file in whatever way suits you. You can use the
+`htpasswd` utility, you can do something like:
+
+```sh
+printf "USER:$(openssl passwd -crypt PASSWORD)\n"
+```
```

Comment:
`$ printf "USER:$(openssl passwd -crypt PASSWORD)\n" >> /path/to/htpasswd`

Reply:
changed to `apr1` though.